### PR TITLE
Update README.md for -fPIC requirement on libevhtp compile.

### PR DESCRIPTION
**Pull request by Martin2112**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Next, we need `libevhtp` version `1.2.10` which is not packaged in Ubuntu yet, s
     wget https://github.com/ellzey/libevhtp/archive/1.2.10.zip
     unzip 1.2.10.zip
     cd libevhtp-1.2.10/
-    cmake -DEVHTP_DISABLE_REGEX:STRING=ON .
+    CFLAGS="-fPIC" cmake -DEVHTP_DISABLE_REGEX:STRING=ON .
```
Comment (AlCutter) [inline README.md]: I did it like this (the the tune of [♪ ♫](https://www.youtube.com/watch?v=kfmtIZKX44s)): `-DCMAKE_C_FLAGS:STRING=-fPIC`.
But I don't know if it makes any real difference

Comment (Martin2112) [inline README.md]: I think they both work. Not sure if either is preferred so you get to pick!

Comment (AlCutter) [inline README.md]: I'd do it my way, then :)

**Comment by Martin2112** [comment]
Done.

**Pull request merged**